version.py

```diff
@@ -4,4 +4,5 @@
 # license that can be found in the LICENSE file or at
 # https://developers.google.com/open-source/licenses/bsd
 
-__version__ = "2.2.0a1"
+# __version__ = "2.2.0a1"
+__version__ = "3.1.0a1"
```

Comment:
This is needed to prevent
```
 django.core.exceptions.ImproperlyConfigured: You must use the latest version of django-spanner 3.1.x with Django 3.1.y (found django-spanner 2.2.0a1).
```